python/ray/tune/examples/wandb_example.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
```

Comment:
we should include this in BUILD

Reply:
also, this example doesn't actually show up on the docs anywhere (though this is a NIT, we can fix later)